testcases/kernel/syscalls/access/access02.c

```diff
@@ -153,13 +155,18 @@ static void verify_access(unsigned int n)
 	access_test(tc, "root");
 
 	/* test as nobody */
+#if 0
```

Comment:
Add a comment here why we are disabling this code block and when we will enable back.
Add a TODO comment so that we can tarck

Reply:
Are we running with nobody user? I thought we have issues running with nobody user that is why we were running with root user. Am I misunderstanding?

Reply:
I didn't uderstand why we are running with nobody user? Are we reverting a change we made before?

Reply:
> Add a comment here why we are disabling this code block and when we will enable back.
> Add a TODO comment so that we can tarck
 
Added the comment

Reply:
> Are we running with nobody user? I thought we have issues running with nobody user that is why we were running with root user. Am I misunderstanding?

Yes, Test runs as nobody user as well. Yes, the test case performs the same sub-test in two user modes namely root and nobody.

Reply:
> I didn't uderstand why we are running with nobody user? Are we reverting a change we made before?

The original test case is designed/implemented to run as nobody user and root user. No, we are not reverting any change.